tests/integration/api_v3/constants/transactionsDryRun.js

```diff
@@ -51,10 +50,29 @@ const TRANSACTION_OBJECT_INVALID = {
 	id: '2fb59d51f7b6517d3008471df13d71015dd174444ee15aa7460e8870a96ebd9c',
 };
 
-const TRANSACTION_ENCODED_VALID = '0a05746f6b656e12087472616e7366657218002080c2d72f2a203972849f2ab66376a68671c10a00e8b8b67d880434cc65b04c6ed886dfa91c2c32340a0804000000000000001080c8afa0251a1402604d9e57a39772fa12f2a860ecf6c1e9cae91122075465737420747828c096b1023a40a7e332b49db6d36c1024a24afbfd6eec2451e41c1cf751a8c609863ffa3e529ef4b8a34cecff29e9fa4bf30de4ce26f58c5d8a06587582a9129a6765064d3404';
+const TRANSACTION_OBJECT_FAIL = {
+	module: 'token',
+	command: 'transfer',
+	fee: '165000',
+	nonce: '0',
+	senderPublicKey: '3972849f2ab66376a68671c10a00e8b8b67d880434cc65b04c6ed886dfa91c2c',
+	signatures: [
+		'451cfcd599ad0b9621b660bea483e5b94c9d0b2a1167295e2bfe92a1fc894f447933cd7e0ef100510c5c9b38d052f5f2503eef83be3b6088fc1c8c09eb9f6d05',
+	],
+	params: {
+		recipientAddress: 'lskjokybqso65y599cbschhzmguoas2uzq2jnnbo2',
+		amount: '10000000000',
+		tokenID: '0400000000000000',
+		data: '',
+	},
+	id: 'dbfc75ea10fc3297213a104079faaa0fc983e3cf0ad05044d5f9d168ff83b172',
+};
+
+const TRANSACTION_ENCODED_VALID = '0a05746f6b656e12087472616e7366657218002080c2d72f2a203972849f2ab66376a68671c10a00e8b8b67d880434cc65b04c6ed886dfa91c2c322f0a0804000000000000001080c8afa0251a1402604d9e57a39772fa12f2a860ecf6c1e9cae9112207546573742074783a40d1ccc2d5fd0cd953b77c608102b7dbb6df7748474667af9e11b6aa050aa56fdc4758b9ae8fad1e0c01c6cc7d5967311e21710c53e0369e711057c09741d75804';
```

Comment:
Please add test cases for `TRANSACTION_ENCODED_INVALID` as well. And, adjust other transactions to implement the exact params as returned in the metadata schema corresponding the module/command, .i.e. remove `accountInitializationFee`.

Reply:
Applied.